src/components/WebexMeetingInfo/WebexMeetingInfo.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import WebexMeetingInfo, {formatMeetingTime} from './WebexMeetingInfo';
+
+jest.mock('../hooks/useMeeting');
+
+describe('Webex Meeting Info component', () => {
+  describe('component snapshots', () => {
```

Comment:
```suggestion
  describe('snapshot', () => {
```

Otherwise it reads `Webex Meeting Info component component snapshots matches...` instead of `Webex Meeting Info component snapshot matches...`